tasks/base_tasks.py

```diff
@@ -584,6 +584,7 @@ class Shp2TempTableTask(TempTableTask):
     '''
 
     encoding = Parameter(default='latin1', significant=False)
+    other_options = ''
```

Comment:
`other_options` couples the usage to the precise `ogr2org` options. Although it might be seen as convenient, in the future it can become a problem not easy to get rid of. As it's only used to override source SRS, what do you think about adding an optional `override_srs` parameter instead, @ethervoid ?